cmake/projects/harfbuzz/hunter.cmake

```diff
@@ -9,6 +9,17 @@ include(hunter_download)
 include(hunter_pick_scheme)
 include(hunter_cmake_args)
 
+hunter_add_version(
```

Comment:
please append the newest version at the end of the list. This makes the diffs of various tools nicer formatted